tests/baselines/reference/functionParameterArityMismatch.errors.txt

```diff
@@ -35,6 +36,6 @@ tests/cases/compiler/functionParameterArityMismatch.ts(14,22): error TS2554: Exp
     ~~~~~~~~~~~~~~~~~
 !!! error TS2575: No overload expects 5 arguments, but overloads do exist that expect either 4 or 6 arguments.
     f2(1, 2, 3, 4, 5, 6, 7);
-                         ~
-!!! error TS2554: Expected 0-6 arguments, but got 7.
-    
```

Comment:
Seems like you lost coverage. Maybe make the new overload take 8 arguments instead?

Reply:
Oops! Thanks @DanielRosenwasser
I force-pushed a new commit and updated my comment above.